plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -452,6 +452,8 @@ public class LibvirtComputingResource extends ServerResourceBase implements Serv
 
     private long _dom0OvercommitMem;
 
+    private int _dom0MinCpuCores;
```

Comment:
@GutoVeronezi - even if it causes mixed style in a class?

Reply:
@mlsorensen, if we keep adding the legacy pattern we will end up increasing the efforts needed to normalize it afterwards. Is not always that we can put efforts on normalize these things; therefore, doing it gradually is the best option.

Reply:
@GutoVeronezi  - then it will never get done, we will just make it more inconsistent and confusing until maybe at some point in the distant future all code is replaced :-)

I have created a PR to refactor all variable names for LibvirtComputingResource, and the same for all classes that were dependent on those variables and also changed. Then I can make the change requested without triggering my OCD by making style consistency worse.

https://github.com/apache/cloudstack/pull/7991

I will refactor this PR, or that PR, depending on which gets merged first.

Thanks

Reply:
Thanks for the initiative, @mlsorensen. I will review that PR as well.

Reply:
thanks @mlsorensen , I can appreciate your approach, even when it would not be mine. Please do as you see fit.